cdap-app-fabric/src/test/java/co/cask/cdap/internal/app/services/http/handlers/ArtifactHttpHandlerTest.java

```diff
@@ -307,6 +307,64 @@ public void testSystemArtifacts() throws Exception {
   }
 
   @Test
+  public void testSystemArtifactIsolation() throws Exception {
```

Comment:
It would be good to have a comment to tell what this method is trying to test (what do mean by isolation is unclear in here).